lib/tasks/cf.rake

```diff
@@ -0,0 +1,12 @@
+namespace :cf do
+  desc "Only run on the first application instance"
+  task on_first_instance: :environment do
+    instance_index = -1
+    begin
+      instance_index = JSON.parse(ENV["VCAP_APPLICATION"])["instance_index"]
+    rescue StandardError
```

Comment:
I wonder what scenario this is wanted in? If you couldn't derive an index and therefore couldn't run migrations wouldn't you rather have a loud crash?

Reply:
I am not sure myself to be honest, should I remove the rescue altogether?
If the JSON is malformed or VCAP_APPLICATION missing, I think indeed we should break.

Otherwise, we might just stop migrating silently.

Reply:
Might be a good place to insert a Sentry call once we have that. Otherwise yeah I'd probably say let's just fail.